src/toolbar/styles/toolbar.m.css

```diff
@@ -0,0 +1,59 @@
+@import '../../common/styles/variables.css';
+
+.actionsFixed {
+	display: flex;
+	flex: 1;
+	justify-content: flex-end;
+}
+
+.menuButtonFixed {
+	position: absolute;
+	right: 0;
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.rootFixed {
+	height: 100%;
+	left: 0;
+	overflow: hidden;
+	position: fixed;
```

Comment:
Does this need to be fixed? It also works with `position: absolute`, and then it isn't constrained to only being document-level (although a toolbar on anything else seems weird, granted).

Reply:
Good catch. The `fixed` styling was left over from the approach that didn't rely on content being a child of the Toolbar.